src/sentry/api/endpoints/project_details.py

```diff
@@ -444,6 +448,11 @@ def put(self, request, project):
                     'sentry:scrub_ip_address',
                     bool(options['sentry:scrub_ip_address']),
                 )
+            if 'sentry:grouping_config' in options:
+                project.update_option(
+                    'sentry:grouping_config',
+                    options['sentry:grouping_config'],
```

Comment:
I intentionally do not validate on set but on read. If the value is not set to a valid value, it falls back to the default. I need to do this anyways because work-in-progress strategies can be removed (and will be).

Reply:
Sounds good.